nucypher/cli/utils.py

```diff
@@ -218,3 +225,47 @@ def deployer_pre_launch_warnings(emitter: StdoutEmitter, etherscan: bool, hw_wal
         emitter.echo(ETHERSCAN_FLAG_ENABLED_WARNING, color='yellow')
     else:
         emitter.echo(ETHERSCAN_FLAG_DISABLED_WARNING, color='yellow')
+
+
+def parse_event_filters_into_argument_filters(event_filters: Tuple) -> Dict:
```

Comment:
A description of the expected input format as part of a docstring will be helpful here for future maintenance.

Reply:
✔️